honeybee_radiance/cli/multiphase.py

```diff
@@ -112,3 +112,87 @@ def view_matrix_command(
         sys.exit(1)
     else:
         sys.exit(0)
+
+
+@multi_phase.command("daylight-matrix")
+@click.argument(
+    "sender-file", type=click.Path(exists=True, file_okay=True, resolve_path=True)
+)
+@click.argument(
+    "receiver-file", type=click.Path(exists=True, file_okay=True, resolve_path=True)
+)
+@click.argument(
+    "octree", type=click.Path(exists=True, file_okay=True, resolve_path=True)
+)
+@click.option("--rad-params", show_default=True, help="Radiance parameters.")
+@click.option(
+    "--rad-params-locked",
+    show_default=True,
+    help="Protected Radiance parameters. "
+    "These values will overwrite user input rad parameters.",
+)
+@click.option(
+    "--output",
+    "-o",
+    show_default=True,
+    help="Path to output file. If a relative path"
+    " is provided it should be relative to project folder.",
+)
+@click.option(
+    "--dry-run",
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="A flag to show the command without running it.",
+)
+def daylight_matrix_command(
+    sender_file,
+    receiver_file,
+    octree,
+    rad_params,
+    rad_params_locked,
+    output,
+    dry_run,
+):
+    """Calculate daylight matrix for a sender file.
+
+    \b
+    Args:
+        sender_file: Path to sender file.
+        receiver_file: Path to receiver file.
+        octree: Path to octree file.
```

Comment:
The sender and receiver file can use a little bit more information in the description to clarify how they look like or what can they be.